lib/logstash/outputs/webhdfs.rb

```diff
@@ -0,0 +1,265 @@
+# encoding: utf-8
+require "logstash/namespace"
+require "logstash/outputs/base"
+require "stud/buffer"
+
+# Summary: Plugin to send logstash events to to files in HDFS via webhdfs 
```

Comment:
Done in https://github.com/dstore-dbap/logstash-output-webhdfs-1/commit/76cd21ccb733834fecb3b21b958a0a440fddcf0a